app/models/host/base.rb

```diff
@@ -326,6 +326,10 @@ def import_missing_ids
     # Provide _id aliases for consistency with the _name methods
     alias_attribute :hardware_model_id, :model_id
 
+    def force_inherited(field_name)
+      nil
+    end
+
```

Comment:
This is on the edge - it's a bit that maybe does not belong to model, I am not sure. If you don't like this approach, I can introduce a controller variable instead.